examples/impute/plot_missing_values.py

```diff
@@ -79,6 +83,13 @@ def get_results(dataset):
     imputer = SimpleImputer(missing_values=0, strategy="mean")
     mean_impute_scores = get_scores_for_imputer(imputer, X_missing, y_missing)
 
+    # Estimate the score after kNN-imputation of the missing values
+    knn_estimator = make_pipeline(
+        KNNImputer(missing_values=0, sample_max_missing=0.99),
```

Comment:
General comment on the whole example: Who abouting indicating missingness as NaN in the entire example, to make it more educational. In some place, having 0 to indicate missingness and replacing by 0 may be confusing to some people.

Reply:
Agreed that missing_values=0 in this example is a bit obscure... but it is a separate issue.